block/src/Pos/Arbitrary/Block.hs

```diff
@@ -402,3 +404,21 @@ instance (Arbitrary SscPayload, HasConfiguration) =>
                 , T.vhpVerifyNoUnknown = not hasUnknownAttributes
                 }
         return . HAndP $ (params, header)
+
+------------------------------------------------------------------------
+-- Pos.Block.Slog.Types
+------------------------------------------------------------------------
+
+instance Arbitrary SlogUndo where
+    arbitrary = genericArbitrary
+    shrink = genericShrink
+
+instance HasConfiguration => Arbitrary Undo where
+    arbitrary = genericArbitrary
+    shrink = genericShrink
+    -- arbitrary = do
```

Comment:
Why is there code that is commented out?

Reply:
my mistake